src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala

```diff
@@ -207,6 +211,23 @@ class PolicyExecutor(
         }
       })
       .toSet
+    if (sinkFilters.sinkType.nonEmpty)
+      matchingSinkIds =
+        matchingSinkIds.filter(ruleCache.getRuleInfo(_).get.id.toLowerCase.contains(sinkFilters.sinkType.toLowerCase))
```

Comment:
Here we are checking on `id` for sinkType, which is invalid

Reply:
I am expecting we will pass sinkType like below:

`sinkType: InternalAPI| ThirdParties.API | SDK | Leakages | Storages`

which will distinguish between all things.

Reply:
Ok, got it. For this to work, we need to have consistent ruleIds across in rules